### ISSUE TITLE
I think we're broken on the Atom upgrade

### ISSUE THREAD

**Issue by dezman** [ahuth/jasmine-runner#4]
```
Uncaught TypeError: this.editor.tokenizedLinesForScreenRows is not a function
```

**Comment by dezman** [comment]
@ahuth hi :)

**Comment by ahuth** [comment]
We've definitely broken, @dezman! Check out https://github.com/ahuth/jasmine-runner/issues/3.

I looked into it briefly but it's not immediately obvious how to do it now. Would love to get it fixed, though. There's not a lot of Jasmine in my life right now, so some help would be great!

**Comment by ahuth** [comment]
Fixed by #5. Thanks for logging this, @dezman!
